components/brave_stats/browser/brave_stats_updater_util.cc

```diff
@@ -30,7 +30,10 @@ std::string GetPlatformIdentifier() {
   else
     return "winx64-bc";
 #elif defined(OS_MAC)
-  return "osx-bc";
+  if (base::SysInfo::CPUModelName().find("Intel") != std::string::npos)
```

Comment:
Maybe we could check something like `ARCH_CPU_ARM64` here?

Reply:
OK, makes sense.

Reply:
tested now and works fine, thanks for the tip